corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -292,6 +292,18 @@ def get_attachments(form_id):
         return list(XFormAttachmentSQL.objects.raw('SELECT * from get_form_attachments(%s)', [form_id]))
 
     @staticmethod
+    def iter_form_ids_by_submission_date(start_datetime, end_datetime):
+        with get_cursor(XFormInstanceSQL) as cursor:
+            cursor.execute(
+                'SELECT * from get_form_ids_by_type_and_date(%s, %s, %s)',
+                [start_datetime, end_datetime, XFormInstanceSQL.NORMAL],
+            )
+            # TODO make this a real iterator
```

Comment:
Something like this is what you want: https://github.com/dimagi/commcare-hq/blob/b7d91efe32231953df9d67097bf600ec3e8bc1f4/custom/hki/management/commands/hki_case_migration.py#L48-L53